chain-abci/tests/abci_app.rs

```diff
@@ -49,6 +49,7 @@ use chain_tx_filter::BlockFilter;
 use hex::decode;
 use kvdb::KeyValueDB;
 use kvdb_memorydb::create;
+use parity_scale_codec::alloc::collections::BTreeSet;
```

Comment:
Is there such a type? Why not just use `std::collections::BTreeSet`?

Reply:
Oh, the IDE did that automatically for me, it doesn't matter, because BTreeSet does not match the requirement.